### ISSUE TITLE
Support of any

### ISSUE THREAD

**Issue by angrymouse** [compilets/compilets#4]
Wondering if supporting any type would be possible with some use of std::any from C++17.
https://en.cppreference.com/w/cpp/utility/any

**Comment by zcbenz** [comment]
Yeah it should be possible, event without it we can still use `std::variant` since the available types of JavaScript is known.

I think eventually I have to add support for `any` to make `JSON.parse` work.
